TestApp/app/screens/AppCenterScreen.js

```diff
@@ -25,7 +25,16 @@ SecretStrings.android.both = `appsecret=${SecretStrings.android.appSecret};${Sec
 
 export default class AppCenterScreen extends Component {
   static navigationOptions = {
-    tabBarIcon: () => <Image style={{ width: 24, height: 24 }} source={DialsTabBarIcon} />
+    tabBarIcon: () => <Image style={{ width: 24, height: 24 }} source={DialsTabBarIcon} />,
+    tabBarOnPress: ({ defaultHandler, navigation }) => {
+      const refreshAppCenterScreen = navigation.getParam('refreshAppCenterScreen');
+
+      // Initial press: the function is not defined yet so nothing to refresh.
+      if (typeof (refreshAppCenterScreen) === 'function') {
```

Comment:
nit - `typeof refreshAppCenterScreen`, not `typeof (refreshAppCenterScreen)`

Reply:
`if (refreshAppCenterScreen)` is shorter and works just fine.

Reply:
resolved